apps/extension/src/components/setting-sheet.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  HamburgerMenuIcon,
+  MixerHorizontalIcon,
+  PersonIcon,
+  SizeIcon,
+} from '@radix-ui/react-icons';
+import { Button, Sheet, SheetContent, SheetFooter, SheetHeader, SheetTrigger } from 'ui/components';
+import { useStore } from '../state';
+import { passwordSelector } from '../state/password';
+import { usePopupNav } from '../utils/navigate';
+import { PopupPath } from '../routes/popup/paths';
+import { PagePath } from '../routes/page/paths';
+
+const links = [
+  {
+    title: 'Settings',
+    icon: <MixerHorizontalIcon className='h-5 w-5 text-foreground' />,
+    href: PopupPath.SETTINGS,
+  },
```

Comment:
I'm not sure why we have a sheet with icons that link to another page with icons. Let's just keep all the icons on the sheet and not link to another page.

Reply:
Let's address this after the designs are finalized